src/main/java/mate/jdbc/dao/impl/DriverDaoImpl.java

```diff
@@ -17,7 +18,7 @@
 public class DriverDaoImpl implements DriverDao {
     @Override
     public Driver create(Driver driver) {
-        String query = "INSERT INTO drivers (name, license_number) VALUES (?, ?)";
+        String query = "INSERT INTO drivers (name, licence_number) VALUES (?, ?)";
```

Comment:
Wrong name of colom "licence_number".
Corect is "license_number".

Reply:
> Wrong name of colom "licence_number". Corect is "license_number".

Licence and license are east to confuse. 'Licence' is not used in the US. In the UK, 'license' is used as a verb, while 'licence' is the noun.  ;)